specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -451,10 +704,33 @@
             "$ref": "#/definitions/PrivateLinkServiceConnection"
           },
           "description": "A grouping of information about the connection to the remote resource. Used when the network admin does not have access to approve connections to the remote resource."
+        },
+        "customDnsConfigs": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CustomDnsConfig"
+          },
+          "description": "A grouping of information about dns configurations connecting to the private endpoint."
         }
       },
       "description": "Properties of the private endpoint."
     },
+    "CustomDnsConfig": {
+      "properties": {
+        "fqdn": {
+          "type": "string",
+          "description": "Fqdn that resolves for private endpoint ip address."
+        },
+        "ipAddresses": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The private ip address of the private endpoint."
```

Comment:
A list of private ip addresses of the private endpoint.

Reply:
done, will push once I'm done with rest of the comments.